examples/cms-datocms/jsconfig.json

```diff
@@ -0,0 +1,5 @@
+{
+  "compilerOptions": {
+    "baseUrl": "."
```

Comment:
We don't suggest this configuration, could we please use absolute imports that start with `@/` instead?

Reply:
@Timer Just using what's documented in the Next.js 9.4 release [blog post](https://nextjs.org/blog/next-9-4#absolute-imports-and-aliases). 

![screenshot](https://dl.dropbox.com/s/qj18cf8239gcnst/Screenshot%202020-05-18%2012.42.38.png?dl=0)

Reply:
We used that example for tersity, but something like this should be more appropriate:
```json
{
  "compilerOptions": {
    "baseUrl": "node_modules",
    "paths": {
      "@/*": ["../*"]
    }
  }
}
```

This prevents you from accidentally shadowing npm modules.

Reply:
@Timer 

I dropped in the snippet you shared, which broke everything. 

```json
{
  "compilerOptions": {
    "baseUrl": "node_modules",
    "paths": {
      "@/*": ["../*"]
    }
  }
}
```
I set the `baseUrl` back to `.` and everything worked again. I went ahead and added three aliases `components`, `lib`, `styles`. 

I've tested this PR here:

💻 https://github.com/gregrickaby/datocms-next-js-blog-demo-2392
👀 https://datocms-next-js-blog-demo-2392.now.sh/


```json
{
  "compilerOptions": {
    "baseUrl": ".",
    "paths": {
      "@/components/*": ["components/*"],
      "@/lib/*": ["lib/*"],
      "@/styles/*": ["styles/*"]
    }
  }
}
```

Let me know if this is what you were thinking.